tools/build

```diff
@@ -2,9 +2,6 @@
 
 set -ex
 
-# TODO After we upgrade to 0.4 make this change in docker/ci/Dockerfile and then remove this line.
-export PATH=/home/postgres/pgx/0.4/bin:$PATH
```

Comment:
We need to remove this in a separate pull request after this one so each commit is safe and clean on its own.

Reply:
Opened #436 to make the changes separate